content/hi/docs/contribute/new-content/open-a-pr.md

```diff
@@ -0,0 +1,34 @@
+---
+title: एक पुल अनुरोध खोलना
+content type: संकल्पना
+weight: 10
+card:
+    name: योगदान देना
+    weight: 40
+---
+
+<!-- अवलोकन  -->
+
+{{< note >}}
+**कोड डेवलपर्स:**:यदि आप किसी के लिए एक नई सुविधा का दस्तावेजीकरण कर रहे हैं
+आगामी कुबेरनेट्स रिलीज, देखना
+[दस्तावेज़ एक नई सुविधा](/docs/contribute/new-content/new-features/)|
+{{< /note >}}
+
+नए सामग्री पृष्ठों में योगदान करने या मौजूदा सामग्री पृष्ठों में सुधार करने के लिए, पुल अनुरोध (पीआर) खोलें। सुनिश्चित करें कि आप [शुरू करने से पहले](/docs/contribute/new-content/overview/#before-you-begin) में सभी आवश्यकताओं का पालन करते हैं|
+
+यदि आपका परिवर्तन छोटा है, या आप git से अपरिचित हैं, तो [GitHub का उपयोग करके परिवर्तन](#changes-using-github) पढ़ें किसी पृष्ठ को संपादित करने का तरीका जानने के लिए।
```

Comment:
```suggestion
यदि आपका परिवर्तन छोटा है, या आप git से अपरिचित हैं, तो किसी पृष्ठ को संपादित करने का तरीका जानने के लिए [GitHub का उपयोग करके परिवर्तन](#changes-using-github) पढ़ें।
```

Reply:
Thank you for the suggestions! I'll add a new commit into the new PR as mentioned above.